packages/web-service/src/pages/submission/submission.js

```diff
@@ -11,10 +13,22 @@ export const checkData = async (request, h) => {
   return null
 }
 
+export const getData = async request => {
+  const { userId, applicationId } = await request.cache().getData()
```

Comment:
Why not just

```
const { applicationId } = await request.cache().getData()
const application = await APIRequests.APPLICATION.getById(applicationId)

```
as used throughout

Reply:
You're right :) 

Resolved and waiting for sonar cube pipeline